modules/capability/go.mod

```diff
@@ -1,18 +1,18 @@
-module github.com/cosmos/ibc-go/modules/capability
+module github.com/cosmos/ibc-go/modules/capability/v7
```

Comment:
The path is not correct. It should resolve to `github.com/cosmos/ibc-go/v7/modules/capability` instead. I am wondering why you need a `go.mod` here at all? The other packets in "module" don't have a `go.mod` and inherit the module path from the root go.mod.

Reply:
I dont think that you're right here, because it is a multi module repository.... I think

And the reason there's a module there is the work on modularization going on in the sdk and also in ibc

Reply:
This project is has `go.mod` files in root and e2e dirs. Everything within `modules/` is within the same `github.com/cosmos/ibc-go/v7` Go module path.
It can make sense to split every subpackage in their own Go module but this should not be part of this PR.
Let's focus on a working example. We should not assume the release cycle of `capability` is different than the rest of the project. Using different Go modules adds quite some complexity to the project.
The SDK uses the `cosmossdk.io` domain and different tags for example to not conflict on the root path.

Reply:
the modularization isn't a part of this PR, it's present in the main branch and the branch we're targeting, and isn't something new to this PR

Reply:
Apologies, you are right with the main branch. Seems that I was looking at an older version. I am a bit lost now. 🤷 
There is a versioned path for the main repo but no versioned path for the capability. I would appreciate some guidance how to integrate the capability now. I  👀  3 different Go paths
* github.com/cosmos/ibc-go/v7
* github.com/cosmos/ibc-go/modules/capability (main)
* github.com/cosmos/ibc-go/modules/capability/v7 (notional)

Reply:
might as well make ours match main, my changes were mainly to see if it solved the issue that you were facing.

Reply:
quick heads up: it resolves with 2 replace statements now. Thanks @faddat !
```
github.com/cosmos/ibc-go/modules/capability => github.com/notional-labs/ibc-go/modules/capability v0.0.0-20230616092119-6e16d6223af4
github.com/cosmos/ibc-go/v7 => github.com/notional-labs/ibc-go/v7 v7.0.0-20230616092119-6e16d6223af4
```